src/components/tools/tool-tile.tsx

```diff
@@ -335,16 +335,21 @@ export class ToolTileComponent extends BaseComponent<IProps, IState> {
     }
   }
 
-  private handleCopyImportJson = () => {
+  private handleCopyImportJsonToClipboard = () => {
     const { appConfig, unit } = this.stores;
     const unitBasePath = appConfig.getUnitBasePath(unit.code);
     const transformImageUrl = (url: string, filename?: string) => {
       return transformCurriculumImageUrl(url, unitBasePath, filename);
     };
     const toolApiInterface = this.context;
     const toolApi = toolApiInterface?.getToolApi(this.modelId);
-    const importJson = toolApi?.exportContentAsTileJson?.({ transformImageUrl });
-    importJson && navigator.clipboard.writeText(importJson);
+    let tileJsonString = toolApi?.exportContentAsTileJson?.({ transformImageUrl });
+    if (tileJsonString) {
+      // Put all exported content in a top-level object, under key: "content"
+      const tileJson = JSON.parse(tileJsonString);
+      tileJsonString = JSON.stringify({ content: tileJson }, null, "  ");
```

Comment:
I should have anticipated this. There's an additional constraint that I didn't mention earlier. Some of the tiles, notably the geometry tile, take pains to produce human-readable/editable JSON. Because this is used for authoring it is anticipated that authors may need to tweak things after export. The problem with `JSON.stringify()` is that it either collapses or explodes all objects/arrays. For readability, sometimes it makes sense to collapse some objects into a single line while allowing others to span multiple lines. For instance, here's a geometry tile export of a single polygon:
```json
{
  "type": "Geometry",
  "title": "Graph 1",
  "board": {
    "properties": {
      "axisMin": [-2.186, -1.093],
      "axisRange": [26.23, 17.486]
    }
  },
  "objects": [
    { "type": "point", "parents": [0, 4.863387978142076], "properties": {"id":"TSHsOr-DshqdI6wh","snapToGrid":true,"snapSizeX":0.1,"snapSizeY":0.1} },
    { "type": "point", "parents": [0, -0.0546448087431694], "properties": {"id":"xucCyMI7hEqdnS5l","snapToGrid":true,"snapSizeX":0.1,"snapSizeY":0.1} },
    { "type": "point", "parents": [4.918032786885246, 4.972677595628415], "properties": {"id":"WM4t7MkqR6S8ZkiC","snapToGrid":true,"snapSizeX":0.1,"snapSizeY":0.1} },
    { "type": "polygon", "parents": ["TSHsOr-DshqdI6wh", "xucCyMI7hEqdnS5l", "WM4t7MkqR6S8ZkiC"],"properties": {"id":"bahmgQGBYrlHRQMN"} }
  ]
}
```
and here's the same export with `JSON.stringify()`:
```json
{
  "type": "Geometry",
  "title": "Graph 1",
  "board": {
    "properties": {
      "axisMin": [
        -2.186,
        -1.093
      ],
      "axisRange": [
        26.23,
        17.486
      ]
    }
  },
  "objects": [
    {
      "type": "point",
      "parents": [
        0,
        4.863387978142076
      ],
      "properties": {
        "id": "TSHsOr-DshqdI6wh",
        "snapToGrid": true,
        "snapSizeX": 0.1,
        "snapSizeY": 0.1
      }
    },
    {
      "type": "point",
      "parents": [
        0,
        -0.0546448087431694
      ],
      "properties": {
        "id": "xucCyMI7hEqdnS5l",
        "snapToGrid": true,
        "snapSizeX": 0.1,
        "snapSizeY": 0.1
      }
    },
    {
      "type": "point",
      "parents": [
        4.918032786885246,
        4.972677595628415
      ],
      "properties": {
        "id": "WM4t7MkqR6S8ZkiC",
        "snapToGrid": true,
        "snapSizeX": 0.1,
        "snapSizeY": 0.1
      }
    },
    {
      "type": "polygon",
      "parents": [
        "TSHsOr-DshqdI6wh",
        "xucCyMI7hEqdnS5l",
        "WM4t7MkqR6S8ZkiC"
      ],
      "properties": {
        "id": "bahmgQGBYrlHRQMN"
      }
    }
  ]
}
```
which takes it from 16 lines to 68 lines and makes it much harder to comprehend the underlying geometry. To preserve the readability your transform should operate on the string(s) directly, i.e. indent each line of the returned string by two spaces and put it in the middle of the wrapper.

Reply:
Here's how it now looks (with the commit that I'm about to push) before and after the 'wrapping' logic:

```
tileJsonString {
  "type": "Geometry",
  "board": {
    "properties": {
      "axisMin": [-1.093, -1.093],
      "axisRange": [26.23, 17.486]
    }
  },
  "objects": [
    { "type": "point", "parents": [3.5478753805263747, 2.622950819672131], "properties": {"id":"a2964875-1a40-4eb7-bb80-68817b105b2d","snapToGrid":true,"snapSizeX":0.1,"snapSizeY":0.1} },
    { "type": "point", "parents": [12.338039314952603, 8.688524590163935], "properties": {"id":"c187c2b6-198f-444c-b444-732155ddabe0","snapToGrid":true,"snapSizeX":0.1,"snapSizeY":0.1} },
    { "type": "point", "parents": [18.998908880169996, 6.830601092896175], "properties": {"id":"b8367afb-01e9-4986-b528-ed19e9fae0b3","snapToGrid":true,"snapSizeX":0.1,"snapSizeY":0.1} },
    { "type": "point", "parents": [21.346592415451536, 13.770491803278688], "properties": {"id":"059a5e1a-3178-4fc3-86cc-a2b8ef5a18d1","snapToGrid":true,"snapSizeX":0.1,"snapSizeY":0.1} },
    { "type": "point", "parents": [21.564981581524236, 10.819672131147541], "properties": {"id":"2cf93204-5c35-4266-904f-32554fc94528","snapToGrid":true,"snapSizeX":0.1,"snapSizeY":0.1} },
    { "type": "point", "parents": [24.45863803198753, 13.060109289617486], "properties": {"id":"d83ef411-888f-4ab7-b759-c8192353375a","snapToGrid":true,"snapSizeX":0.1,"snapSizeY":0.1} },
    { "type": "movableLine", "parents": [{ "type": "point", "parents": [1, 12.283442023434429] }, { "type": "point", "parents": [1, 23.585081367696723] }],"properties": {"id":"02fff56a-b656-427a-8b2e-366d11a73e69"} }
  ]
}
END
```

```
tileJsonString {
  "content": {
    "type": "Geometry",
    "board": {
      "properties": {
        "axisMin": [-1.093, -1.093],
        "axisRange": [26.23, 17.486]
      }
    },
    "objects": [
      { "type": "point", "parents": [3.5478753805263747, 2.622950819672131], "properties": {"id":"a2964875-1a40-4eb7-bb80-68817b105b2d","snapToGrid":true,"snapSizeX":0.1,"snapSizeY":0.1} },
      { "type": "point", "parents": [12.338039314952603, 8.688524590163935], "properties": {"id":"c187c2b6-198f-444c-b444-732155ddabe0","snapToGrid":true,"snapSizeX":0.1,"snapSizeY":0.1} },
      { "type": "point", "parents": [18.998908880169996, 6.830601092896175], "properties": {"id":"b8367afb-01e9-4986-b528-ed19e9fae0b3","snapToGrid":true,"snapSizeX":0.1,"snapSizeY":0.1} },
      { "type": "point", "parents": [21.346592415451536, 13.770491803278688], "properties": {"id":"059a5e1a-3178-4fc3-86cc-a2b8ef5a18d1","snapToGrid":true,"snapSizeX":0.1,"snapSizeY":0.1} },
      { "type": "point", "parents": [21.564981581524236, 10.819672131147541], "properties": {"id":"2cf93204-5c35-4266-904f-32554fc94528","snapToGrid":true,"snapSizeX":0.1,"snapSizeY":0.1} },
      { "type": "point", "parents": [24.45863803198753, 13.060109289617486], "properties": {"id":"d83ef411-888f-4ab7-b759-c8192353375a","snapToGrid":true,"snapSizeX":0.1,"snapSizeY":0.1} },
      { "type": "movableLine", "parents": [{ "type": "point", "parents": [1, 12.283442023434429] }, { "type": "point", "parents": [1, 23.585081367696723] }],"properties": {"id":"02fff56a-b656-427a-8b2e-366d11a73e69"} }
    ]
  }
}
END
```

(Note: the "END" suffix, and "tileJsonString " prefix are for debugging purposes, and are NOT part of the JSON string output.)